flink/src/test/java/org/apache/sedona/flink/FunctionTest.java

```diff
@@ -265,6 +265,27 @@ public void testIsEmpty() {
         assertEquals(false, result);
     }
 
+    @Test
+    public void testX() {
```

Comment:
Can you use the same test in testZ for testX and testY?

Reply:
Updated the PR. Thanks for the review!